server/src/main/resources/routes

```diff
@@ -3,7 +3,10 @@
 # ~~~~
 
 # API routes
--> /api/v1  de.frosner.broccoli.ApiV1Router
+-> /api/v1  de.frosner.broccoli.routes.ApiV1Router
+
+# Downloads
+-> /downloads/ de.frosner.broccoli.routes.DownloadsRouter
```

Comment:
It does not send JSON and it does not receive JSON. It receives a GET request and returns a text. But the format does not determine whether it is part of an API or not. For me there are two types of endpoints:

1. Endpoints that are meant to be accessed by a browser, serving a webpage with a layout, etc. This is the index.html basically.
2. Endpoints that are meant to be accessed by programs, CLIs, scripts. This is useful for interacting with Broccoli without using a browser.

While we are certainly using the endpoint through a browser this is just temporary. We do it because we didn't include the logs directly in the UI. But what you get there is a resource and the protocol should be versioned. Why do I care? If someone uses Broccoli from a script or from the command line and we change the API (e.g. no longer return text but JSON), then it should be reflected in a new version of the API.

I agree that not sending the logs in JSON is a bit inconsistent. But we can make that call. We just need to document it.

I didn't fully get the separate URL prefix issue. Each endpoint can return 404 individually. If you request an instance that does not exist, you get 404. Same for logs.